packages/jest-cli/src/jest.js

```diff
@@ -50,17 +51,11 @@ const KEYS = {
   U: '75',
 };
 
-const printHeader = pipe => {
-  if (process.stdout.isTTY) {
-    pipe.write(chalk.bold.dim('Determining test suites to run...'));
-  }
-};
-
 const getMaxWorkers = argv => {
   if (argv.runInBand) {
     return 1;
   } else if (argv.maxWorkers) {
-    return argv.maxWorkers;
+    return parseInt(argv.maxWorkers, 10);
```

Comment:
hahaha. How long did it take you to figure out that you need to do this so that throat won't explode? Me, in the past when I did experiments, I had to sleep over it.